src/test/kotlin/org/objectionary/ddr/TestBase.kt

```diff
@@ -73,8 +83,30 @@ interface TestBase {
 
     /**
      * @return name of the test being executed
+     * @throws TestNotFoundException
+     */
+    fun getTestName(): String {
+        Thread.currentThread().stackTrace.forEach {
+            val methodName = it.methodName
```

Comment:
@c71n93 We are trying to avoid compound names. According to: https://www.yegor256.com/2015/01/12/compound-name-is-code-smell.html